config/lti_dev_key_sample.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Canvas Course Manager",
+  "scopes": [],
+  "extensions": [
+    {
+      "platform": "canvas.instructure.com",
+      "settings": {
+        "platform": "canvas.instructure.com",
+        "default": "visible",
+        "placements": [
+          {
+            "placement": "course_navigation",
+            "message_type": "LtiResourceLinkRequest",
+            "target_link_uri": "https://{ccm_app_url}"
```

Comment:
You need to add `"visibility": "admins"` since this tool should only be visibility for admins in a course

Reply:
Done.